libs/accounts/recovery-phone/src/lib/recovery-phone.service.ts

```diff
@@ -94,6 +95,18 @@ export class RecoveryPhoneService {
       }
     }
 
+    // Rejects the phone number if it has been registered for too many accounts
+    const countByPhoneNumber =
+      await this.recoveryPhoneManager.getCountByPhoneNumber(phoneNumber);
+
+    if (
+      (this.config.maxRegistrationsPerNumber &&
+        countByPhoneNumber >= this.config.maxRegistrationsPerNumber) ||
+      countByPhoneNumber > 4
```

Comment:
What's the reason to have > 4 condition here when we have a configured value already?

Reply:
This made sense late last night, not so much after a few cups of coffee this morning. Will remove